packages/integration-karma/test/wire/wiring/index.spec.js

```diff
@@ -211,6 +213,27 @@ describe('wiring', () => {
                     }, 5);
                 });
         });
+
+        it('should not trigger update when api value is re-set', done => {
+            const elm = createElement('x-same-config', { is: WireWithApi });
+            elm.a = 5;
+
+            const spy = [];
+            EchoWireAdapter.setSpy(spy);
+
+            setTimeout(() => {
+                expect(spy.length).toBe(1); // first update
+                expect(elm.result).toBe(5);
+
+                elm.a = 5;
+
+                setTimeout(() => {
+                    expect(spy.length).toBe(1); // no extra call with the config
```

Comment:
```suggestion
                spy.length = 0;
                elm.a = 5;

                setTimeout(() => {
                    expect(spy.length).toBe(0); // no extra call with the config
```

Reply:
@ravijayaramappa the 1 call is the initial config call, that one is always executed.